WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterViewModel.cs

```diff
@@ -187,6 +188,10 @@ private async Task OnDoTransactionBroadcastAsync()
 				SetSuccessMessage("Transaction is successfully sent!");
 				TransactionString = "";
 			}
+			catch (PSBTException ex)
```

Comment:
Why this catch? Why not let the exception flow? The same result.

Reply:
@nopara73 because if there is too much error, the PSBTException is fucking huge and unreadable.

Reply:
@nopara73 if I let the exception fall through, what happens is this:
![image](https://user-images.githubusercontent.com/16554748/64230355-3d3f3880-cf1f-11e9-97c2-7cc2def8a274.png)

Reply:
Makes sense.